compiler/test/dotc/run-test-pickling.blacklist

```diff
@@ -38,6 +38,7 @@ mixins1
 outerPatternMatch
 paramForwarding_separate
 quote-and-splice
+quote-change-owner
```

Comment:
why is this blacklisted?

Reply:
Because it used separate compilation. I'm getting a bit annoyed by this blacklist.

Reply:
Ah, I see.

Reply:
I may have a simple solution for it. I will implement it in an other PR